resources/services/servicebus/namespaces.go

```diff
@@ -66,166 +66,373 @@ func ServicebusNamespaces() *schema.Table {
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("ID"),
+			},
+			{
+				Name:        "name",
+				Description: "Resource name",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "type",
+				Description: "Resource type",
+				Type:        schema.TypeString,
+			},
 		},
 		Relations: []*schema.Table{
 			{
 				Name:          "azure_servicebus_namespace_private_endpoint_connections",
-				Description:   "List of private endpoint connections.",
-				Resolver:      fetchServicebusNamespacesPrivateEndpointConnections,
+				Description:   "PrivateEndpointConnection properties of the PrivateEndpointConnection",
+				Resolver:      fetchServicebusNamespacePrivateEndpointConnections,
 				IgnoreInTests: true,
 				Columns: []schema.Column{
 					{
 						Name:        "namespace_cq_id",
-						Description: "Unique ID of azure_servicebus_namespaces table (FK)",
+						Description: "Unique CloudQuery ID of azure_servicebus_namespaces table (FK)",
 						Type:        schema.TypeUUID,
 						Resolver:    schema.ParentIdResolver,
 					},
+					{
+						Name:        "private_endpoint_id",
+						Description: "The ARM identifier for Private Endpoint",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("PrivateEndpointConnectionProperties.PrivateEndpoint.ID"),
+					},
+					{
+						Name:        "status",
+						Description: "Status of the connection",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("PrivateEndpointConnectionProperties.PrivateLinkServiceConnectionState.Status"),
+					},
+					{
+						Name:        "status_description",
+						Description: "Description of the connection state",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("PrivateEndpointConnectionProperties.PrivateLinkServiceConnectionState.Description"),
+					},
+					{
+						Name:        "provisioning_state",
+						Description: "Provisioning state of the Private Endpoint Connection",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("PrivateEndpointConnectionProperties.ProvisioningState"),
+					},
+					{
+						Name:        "system_data",
+						Description: "The system meta data relating to this resource",
+						Type:        schema.TypeJSON,
+					},
 					{
 						Name:        "id",
-						Description: "Resource Id.",
+						Description: "Resource Id",
 						Type:        schema.TypeString,
 						Resolver:    schema.PathResolver("ID"),
 					},
 					{
 						Name:        "name",
-						Description: "Resource name.",
+						Description: "Resource name",
 						Type:        schema.TypeString,
 					},
 					{
 						Name:        "type",
-						Description: "Resource type.",
+						Description: "Resource type",
 						Type:        schema.TypeString,
 					},
+				},
+			},
+			{
+				Name:        "azure_servicebus_namespace_topics",
+				Description: "SBTopic description of topic resource",
```

Comment:
I guess maybe a name got prefixed to all the table descriptions? I would drop `SBTopic` from the start of this and/or reword it slightly:
```suggestion
				Description: "Description of servicebus namespace topic resource",
```